docs/README.md

```diff
@@ -85,7 +85,7 @@ Coder is free and open source under
 [GNU Affero General Public License v3.0](https://github.com/coder/coder/blob/main/LICENSE).
 All developer productivity features are included in the Open Source version of
 Coder.
-A [Premium license is available](https://coder.com/pricing#compare-plans) for enhanced
+An [Enterprise license is available](https://coder.com/pricing#compare-plans) for enhanced
```

Comment:
We want to change Enterprise to Premium throughout the docs. Can you confirm @stirby?

Reply:
you're totally right and I can't remember why I edited this back. fixing now